explorer/app/components/StaticImage/StaticImage.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @next/next/no-img-element */
+
+/**
+ * This component should be used only for images from the /images folder. And the url should be
+ * a relative path from /images
+ * These images will be optimized at the build time by next-optimized-images
+ */
+import React from "react";
+
+export interface StaticImageProps {
+  src: string;
+  width?: number;
+  height?: number;
+  alt: string;
+}
+
+export const StaticImage: React.FC<StaticImageProps> = ({
```

Comment:
Can you add a basic overview of how this works to the repo README? For example, what packages are required, what the config params are and why are they required, why is the `required` required.

Reply:
I've changed the component API. If I use ES modules instead of CommonJS the images will just work on storybook. 
Also, on second thought it seems a better API for the component, making it clearer that you need to pass an imported image instead of a string formated in a specific way.
I've updated the Readme. Let me know if it makes sense :)